### PR TITLE
Fixed

### PR DESCRIPTION
**Pull request by MeirShpilraien**
1. No any spacific support from RedisGears is required now. This client use rg.pyexecute
2. Match signitures of 'run' and 'register' functions
3. Added repartition step

### CODE REVIEW

**Comment by MeirShpilraien** [comment]
- [x] Add local aggregate and local aggregate by

File: gearsclient/redisgears_builder.py [hunk 1]
```diff
@@ -250,20 +250,59 @@ def limit(self, count, offset=0):
...
+    return redisExecute(*args)
+
+def hashtag():
+    from redisgears import getMyHashTag as redisHashtag
+    return redisHashtag()
+
+class atomic:
+    def __init__(self):
+        from redisgears import atomicCtx as redisAomic
+        self.atomic = redisAomic()
```
Comment (rafie) [inline gearsclient/redisgears_builder.py]: Aomic -> Atomic

**Comment by MeirShpilraien** [comment]
@manueljgarciar I know you used this RedisGears client before. Those are some of the changes I talked to you about here https://github.com/RedisGears/redisgears-py/pull/6. If you want to take a look, feedback is appreciated.

**Review by rafie — COMMENTED** [review]
Looks cool. One small typo.

**Pull request merged**